### PR TITLE
[13.0][MIG] hr_contract_operating_unit

### PR DESCRIPTION
**Pull request by nikul-serpentcs**

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@JordiBForgeFlow @AaronHForgeFlow Can you please take a look?

File: hr_contract_operating_unit/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Eficent Business and IT Consulting Services S.L. <contact@eficent.com>
```
Comment (AaronHForgeFlow) [inline hr_contract_operating_unit/readme/CONTRIBUTORS.rst]: Hi @nikul-serpentcs could you put ForgeFlow S.L. instead? thanks in advance

File: hr_contract_operating_unit/tests/test_hr_contract_operating_unit.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Copyright 2016-19 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline hr_contract_operating_unit/tests/test_hr_contract_operating_unit.py:1]: also here ForgeFlow S.L.

**Review by AaronHForgeFlow — APPROVED** [review]
Tested LGTM!

**Comment by nikul-serpentcs** [comment]
@AaronHForgeFlow @JordiBForgeFlow Ready to merge

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by nikul-serpentcs** [comment]
@AaronHForgeFlow Ready to merge

**Pull request merged**